src/main/java/net/sf/jsqlparser/statement/select/GroupByElement.java

```diff
@@ -9,21 +9,22 @@
  */
 package net.sf.jsqlparser.statement.select;
 
-import net.sf.jsqlparser.expression.Expression;
-import net.sf.jsqlparser.expression.operators.relational.ExpressionList;
-import net.sf.jsqlparser.expression.operators.relational.ParenthesedExpressionList;
-
 import java.io.Serializable;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.Collections;
 import java.util.List;
 import java.util.Optional;
+import net.sf.jsqlparser.expression.Expression;
+import net.sf.jsqlparser.expression.operators.relational.ExpressionList;
+import net.sf.jsqlparser.expression.operators.relational.ParenthesedExpressionList;
 
 public class GroupByElement implements Serializable {
     private ExpressionList groupByExpressions = new ExpressionList();
     private List<ExpressionList> groupingSets = new ArrayList<>();
+    // postgres rollup is an ExpressionList
```

Comment:
based on https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-rollup/
I understand, that Postgres has no `ROLLUP` clause but instead uses an Aggregate Function?

From JSQLParser's view this means:

1) for MySQL, implement the `ROLLUP` keyword
2) for Postgres, do nothing because `Rollup()` would be just another Expression in the `GROUP BY` expression list

Reply:
ok, I think I misunderstood based on the comment:

1) you implement `hasMySQLStyleRollup`
2) which is not needed for Postgres, since Postgres uses an Aggregate Function

Reply:
Since this PR is solving the specified problem, but someone may want to implement postgresql later, it will be called mysqlwithrollup. i considered implementing postgresql at the same time, but the differences are just too great for a simple unification.